packages/app-toolbox/src/Unlock.tsx

```diff
@@ -129,6 +129,7 @@ class Unlock extends React.PureComponent<Props, State> {
             <Password
               className='medium'
               isError={!!unlockError}
+              help={t('The account password as was specified in the creation of this account.')}
```

Comment:
I'm not native but that sounded a bit weird. Gave my best shot.
```suggestion
              help={t('The account\'s password specified at the creation of this account.')}
```